tmail-backend/apps/distributed/src/main/java/com/linagora/tmail/james/app/DistributedJamesConfiguration.java

```diff
@@ -19,7 +19,7 @@
 import org.apache.james.vault.VaultConfiguration;
 
 import com.github.fge.lambdas.Throwing;
-import com.linagora.tmail.blob.blobid.list.BlobStoreConfiguration;
+import com.linagora.tmail.blob.blobguice.BlobStoreConfiguration;
```

Comment:
```suggestion
import com.linagora.tmail.blob.guice.BlobStoreConfiguration;
```

?

(Repeting blob twice looks weird